deepinv/transform/projective.py

```diff
@@ -0,0 +1,330 @@
+from dataclasses import dataclass
+from typing import Union
+
+import numpy as np
+import torch
+from PIL import Image
+
+from scipy.spatial.transform import Rotation
```

Comment:
Do you think it is possible to remove this dependency?
If I understood well, it is used for generating a rotation matrix from euclidean angles on a single line of code.

Reply:
Done!